install/gcp/installer/util/utils.py

```diff
@@ -109,20 +109,50 @@ def _print_banner(border_symbol, edge_symbol, corner_symbol,
 
 
 def get_forseti_version():
-    """Get Forseti version from version file.
+    """Get the current Forseti version. If the current version is a branch
+    or a tag, we will return it. Otherwise we will read the __init__ file
+    for the latest version.
 
     Returns:
         str: The version.
     """
     version = None
+
+    # Check version by tag.
+    return_code, out, err = run_command(
+        ['git', 'describe', '--tags', '--exact-match'])
+    # The git command above will return the tag name if we checked out
+    # a tag, will throw an exception otherwise.
+    if return_code:
+        print(err)
+    else:
+        return 'tags/{}'.format(out.strip())
+
+    # Check version by branch.
+    return_code, out, err = run_command(
+        ['git', 'symbolic-ref', '-q', '--short', 'HEAD'],
+        number_of_retry=0)
+    # The git command above will return empty if the user is not currently
+    # on a branch.
+    if return_code:
+        print(err)
+    else:
+        version = out.strip()
+
+    if version and version != 'stable':
+        # If there is a version and it's not branch `stable`, we don't need
+        # to read the __init__.py file for version.
+        return version
+
+    # Check version by source code.
```

Comment:
Add a comment to say that install from stable branch must be pinned to a release tag.
```
# Installing from stable branch must be pinned to a release tag.
```

Reply:
Updated.